src/GraphUtils/utils.js

```diff
@@ -1,3 +1,29 @@
+/**
+ * Get subgroup links from link
+ * @param {array} link - array of links
```

Comment:
this is picky but usually if something is an array it feels like it should be plural, so `links` instead of `link`

Reply:
The code seems like `link` is not an array but an object though?